hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/plan/generators/BaseHoodieCompactionPlanGenerator.java

```diff
@@ -133,25 +132,23 @@ public HoodieCompactionPlan generateCompactionPlan(String compactionInstant) thr
         .getLatestFileSlicesStateless(partitionPath)
         .filter(slice -> filterFileSlice(slice, lastCompletedInstantTime, fgIdsInPendingCompactionAndClustering, instantRange))
         .map(s -> {
-          List<HoodieLogFile> logFiles = s.getLogFiles()
-              // ==============================================================
-              // IMPORTANT
-              // ==============================================================
-              // Currently, our filesystem view could return a file slice with pending log files there,
-              // these files should be excluded from the plan, let's say we have such a sequence of actions
-
-              // t10: a delta commit starts,
-              // t20: the compaction is scheduled and the t10 delta commit is still pending, and the "fg_10.log" is included in the plan
-              // t25: the delta commit 10 finishes,
-              // t30: the compaction execution starts, now the reader considers the log file "fg_10.log" as valid.
-
-              // for both OCC and NB-CC, this is in-correct.
-              .filter(logFile -> completionTimeQueryView.isCompletedBefore(compactionInstant, logFile.getDeltaCommitTime()))
-              .sorted(HoodieLogFile.getLogFileComparator()).collect(toList());
-          if (logFiles.isEmpty()) {
-            // compaction is not needed if there is no log file.
-            return null;
-          }
+          // ==============================================================
+          // IMPORTANT
+          // ==============================================================
+          // Currently, our filesystem view could return a file slice with pending log files there,
+          // these files should be excluded from the plan, let's say we have such a sequence of actions
+
+          // t10: a delta commit starts,
+          // t20: the compaction is scheduled and the t10 delta commit is still pending, and the "fg_10.log" is included in the plan
+          // t25: the delta commit 10 finishes,
+          // t30: the compaction execution starts, now the reader considers the log file "fg_10.log" as valid.
+
+          // for both OCC and NB-CC, this is in-correct.
```

Comment:
The reason why I modified the indentation of the above comment is that mvn:checkstyle determined that the above indentation was incorrect:
<img width="1369" height="55" alt="image" src="https://github.com/user-attachments/assets/4fa31d31-4915-4ad8-928a-745b91f52d00" />